aries-backchannels/acapy/acapy_backchannel.py

```diff
@@ -561,10 +599,71 @@ async def handle_did_exchange_POST(self, op, rec_id=None, data=None):
         if resp_status == 200: resp_text = self.agent_state_translation(op["topic"], operation, resp_text)
         return (resp_status, resp_text)
 
+    async def handle_issue_credential_v2_POST(self, op, rec_id=None, data=None):
+        operation = op["operation"]
+        topic = op["topic"]
+
+        if rec_id is None:
+            agent_operation = self.TopicTranslationDict[topic] + self.issueCredentialv2OperationTranslationDict[operation]
+        else:
+            # if (operation == "send-offer" 
+            #     or operation == "send-request"
+            #     or operation == "issue"
+            #     or operation == "store"
+            # ):
+            # swap thread id for cred ex id from the webhook
+            cred_ex_id = await self.swap_thread_id_for_exchange_id(rec_id, "credential-msg", "cred_ex_id")
+            agent_operation = self.TopicTranslationDict[topic] + "records/" + cred_ex_id + "/" + self.issueCredentialv2OperationTranslationDict[operation]
+            # Make Special provisions for revoke since it is passing multiple query params not just one id.
+            # elif (operation == "revoke"):
+            #     cred_rev_id = rec_id
+            #     rev_reg_id = data["rev_registry_id"]
+            #     publish = data["publish_immediately"]
+            #     agent_operation = self.TopicTranslationDict[topic] + self.issueCredentialv2OperationTranslationDict[operation] + "?cred_rev_id=" + cred_rev_id + "&rev_reg_id=" + rev_reg_id + "&publish=" + str(publish).lower()
+            #     data = None
+            # else:
+            #     agent_operation = acapy_topic + operation
+        
+        log_msg(agent_operation, data)
+        
+        (resp_status, resp_text) = await self.admin_POST(agent_operation, data)
+
+        log_msg(resp_status, resp_text)
+        # Looks like all v2 states are RFC states. Yah!
+        #if resp_status == 200: resp_text = self.agent_state_translation(topic], None, resp_text)
+        resp_text = self.move_state_to_top_level(resp_text)
+        return (resp_status, resp_text)
+
+    def check_aip_version(self, data=None):
+        # check if the data contains the aip_version. If so retrive it and set it localy, then remove it from data
+        if data and "aip_version" in data:
+            self.aip_version = data["aip_version"]
+            data.pop("aip_version")
+            if len(data) == 0:
+                data = None
+        else:
+            self.aip_version = "AIP10"
```

Comment:
Per discussion last week, AIP shouldn't really be a setting on this, but rather the version of the credential exchange protocols being used (issue-credential/v2.0 and present-proof/v2.0).  The RFC being used could also be used as major versions also mean new RFCs.

Reply:
Yes, good catch. That method is left over from the previous approach I was implementing. It is not being used anymore and should be removed. I'll do that.